packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -216,28 +275,41 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         className={mergeStyles({ width: `calc(100% - ${menuButtonContainerStyle.width})`, alignItems: 'center' })}
       >
         {avatar}
-        {me && <Text className={meTextStyle}>{isMeText}</Text>}
+        {me && <Text className={meTextStyle}>{strings.isMeText}</Text>}
         <Stack horizontal className={mergeStyles(infoContainerStyle)}>
           {onRenderIcon && onRenderIcon(props)}
         </Stack>
       </Stack>
-      {menuItems && menuItems.length > 0 && (
-        <>
-          {menuButton}
-          <ContextualMenu
-            items={menuItems}
-            hidden={menuHidden}
-            target={containerRef}
-            onItemClick={onDismissMenu}
-            onDismiss={onDismissMenu}
-            directionalHint={DirectionalHint.bottomRightEdge}
-            className={contextualMenuStyle}
-            calloutProps={{
-              preventDismissOnEvent
-            }}
-          />
-        </>
-      )}
+      {/* Show the participant state for a remote participant only. Hide it on hover. */}
+      {
+        /* @conditional-compile-remove(one-to-n-calling) */
+        /* @conditional-compile-remove(PSTN-calls) */
+        !me && participantStateString && (
+          <div onMouseEnter={() => setItemHovered(true)} style={{ display: itemHovered ? 'none' : 'block' }}>
```

Comment:
should also be hidden when focused on by tab naviation? Or when tapped on mobile?

Reply:
Actually if it's hidden on focus, that could be a a11y issue

Reply:
Spelling out the issues here:

* For keyboard nav users, tabbing onto the "Connecting..." state will hide the string and so the screen reader will not be able to announce the state. A solution is to update the aria string for the participant name to include the connection state (so they tab onto the name and *that* announces the state)

* Verify behavior on mobile. I think this is not an issue because on mobile, there is no overflow button. User needs to tap on the entire menuitem to open the sub-drawer:

![image](https://user-images.githubusercontent.com/82062616/183612050-f9f65b48-d0d5-429b-a263-87f6093594bb.png)
vs
![image](https://user-images.githubusercontent.com/82062616/183612128-094984d9-da66-4f95-ad53-d182c3d9d466.png)

Reply:
Resolving by disabling the menu when a participant is in connecting state